distribution/integ-test-zip/build.gradle

```diff
@@ -35,5 +37,14 @@ publishing {
   }
 }
 
+Task integTestNetty4 = tasks.create("integTestNetty4", RestIntegTestTask) {
```

Comment:
Thanks @spinscale for the suggestion to get the REST tests working inside of transport-netty4.